de.metas.business/src/main/java/de/metas/pricing/service/ProductPrices.java

```diff
@@ -122,16 +132,6 @@ public static void assertMainProductPriceIsNotDuplicate(final I_M_ProductPrice p
 
 	public static void assertUomConversionExists(final I_M_ProductPrice productPrice)
 	{
-		if (productPrice == null || !productPrice.isActive())
```

Comment:
i think it's not unnecessary. you never know who will call your `public static` method